tests/sans2d_vacuum_collision_avoidance.py

```diff
@@ -93,294 +110,65 @@ class Sans2dVacCollisionAvoidanceTests(unittest.TestCase):
     def setUp(self):
         self.ca = ChannelAccess(device_prefix="MOT", default_timeout=30)
```

Comment:
Could you add `default_wait_time=0` and remove all the zero length sleeps throughout?